packages/build-utils/src/get-project-paths.ts

```diff
@@ -0,0 +1,63 @@
+import { detectFramework } from './detect-framework';
+import { DetectorFilesystem } from './detectors/filesystem';
+import frameworks from '@vercel/frameworks';
+
+const MAX_DEPTH_TRAVERSE = 3;
+
+export interface GetProjectPathsOptions {
+  fs: DetectorFilesystem;
+  path?: string;
+  skipPaths?: string[];
+}
+
+export type ProjectPath = string;
+
+export const getProjectPaths = async ({
+  fs,
+  path,
+  skipPaths,
+}: GetProjectPathsOptions): Promise<ProjectPath[]> => {
+  const allPaths: Array<ProjectPath> = [];
+  for (
+    let currentDepth = 0;
+    currentDepth < MAX_DEPTH_TRAVERSE;
+    currentDepth++
+  ) {
```

Comment:
This appears to be unused?

Reply:
I changed this to depth being sent in because it was looping through all directories